.gitignore

```diff
@@ -1,2 +0,0 @@
-.DS_Store
```

Comment:
DO NOT TOUCH .gitignore
I made this file so we would no longer have .DS_Store on our server - see issue #338. Deleting this will cause this issue to reoccur. In other words, DON'T DELETE IT!